docs/en/week13/13-3.md

```diff
@@ -0,0 +1,310 @@
+---
+lang-ref: ch.13-3
+title: Graph Convolutional Network
+authors: Go Inoue, Muhammad Osama Khan, Muhammad Shujaat Mirza, Muhammad Muneeb Afzal
+date: 5 May 2020
+---
+
+## Introduction to Graph Convolutional Network (GCN) 
+
+Graph Convolutional Network (GCN) is one type of architecture that utilizes the structure of data.
+Before going into details, let's have a quick recap on self-attention, as GCN and self-attention are conceptually relevant. 
+
+### Recap: Self-attention
+
+- In self-attention, we have a set of input $\{\boldsymbol{x}_{i}\}^{t}_{i=1}$.
+Unlike a sequence, it does not have an order.
+- Hidden vector $\boldsymbol{h}$ is given by linear combination of the vectors in the set.
+- We can express this as $\boldsymbol{X}\boldsymbol{a}$ using matrix vector multiplication, where $\boldsymbol{a}$ contains coefficients that scale the input vector $\boldsymbol{x}_{i}$.
+
+*For a detailed explanation, refer to the notes of [Week 10]({{site.baseurl}}/en/week10/10-3/).*
+
+### Notation
+
+<center>
+<img src="{{site.baseurl}}/images/week13/13-3/figure1.png" height="200px" /><br>
+<b>Fig. 1</b>: Graph Convolutional Network
+</center>
+
+In Figure 1, vertex $v$ is comprised of two vectors: input $\boldsymbol{x}$ and its hidden representation $\boldsymbol{h}$.
+We also have multiple vertices $v_{j}$, which is comprised of $\boldsymbol{x}_{j}$ and $\boldsymbol{h}_{j}$.
+In this graph, vertices are connected with directed edges.
+
+We represent these directed edges with adjacency vector $\boldsymbol{a}$, where each element $\alpha_{j}$ is set to $1$ if there is a directed edge from $v_{j}$ to $v$.
+
+$$
+\alpha_{j} \stackrel{\tiny \downarrow}{=} 1 \Leftrightarrow v_{j} \rightarrow v
+\tag{Eq. 1}
+$$
+
+The degree (number of incoming edges) $d$ is defined as the norm of this adjacency vector, i.e. $\|\|\boldsymbol{a}\|\|_{1} $, which is the number of ones in the vector $\boldsymbol{a}$.
```

Comment:
The norm is obtained using `\Vert`, the absolute value with `\vert`, and the conditional bar with `\mid`.
Please, remove any `||`, because they are interpreted as a Markdown table.